### ISSUE TITLE
Unexpected label ***

### ISSUE THREAD

**Issue by huayong** [cdmh/deeplab-public#2]
hi all,

   I have compiled the deeplab code successfully. But when I run  the command './run_pascal.sh ', the console printed 
"I0518 15:05:52.635113  7739 net.cpp:219] Network initialization done.
I0518 15:05:52.635121  7739 net.cpp:220] Memory required for data: 7773188176
I0518 15:05:52.635254  7739 solver.cpp:41] Solver scaffolding done.
I0518 15:05:52.635268  7739 solver.cpp:160] Solving vgg128_noup
I0518 15:05:52.635275  7739 solver.cpp:161] Learning Rate Policy: step
F0518 15:05:54.901671  7739 softmax_loss_layer.cpp:85] Unexpected label 255
*** Check failure stack trace: ***
    @     0x7fb4f054edaa  (unknown)
    @     0x7fb4f054ece4  (unknown)
    @     0x7fb4f054e6e6  (unknown)
    @     0x7fb4f0551687  (unknown)
    @           0x51ff43  caffe::SoftmaxWithLossLayer<>::Forward_cpu()
    @           0x57e536  caffe::Net<>::ForwardFromTo()
    @           0x57e977  caffe::Net<>::ForwardPrefilled()
    @           0x47fd6f  caffe::Solver<>::Solve()
    @           0x4199b2  train()
    @           0x4124a1  main
    @     0x7fb4edd73ec5  (unknown)
    @           0x4183e7  (unknown)
    @              (nil)  (unknown)
Aborted (core dumped)"

  But I could ensure that every pixel of each png picture is not greater than 20. From the "softmax_loss_layer.cpp" code, this error showed that there was a pixel label was greater than channels(20).
  And any help would be appreciate.

**Comment by cdmh** [comment]
I forked this repo some time ago and the source has now disappeared from GitHub.
I never used the code, so cannot help.

Sorry.
